src/nls/root/strings.js

```diff
@@ -67,6 +67,7 @@ define({
     "ERROR_DELETING_FILE"               : "An error occurred when trying to delete the {2} <span class='dialog-filename'>{0}</span>. {1}",
     "INVALID_FILENAME_TITLE"            : "Invalid {0}",
     "INVALID_FILENAME_MESSAGE"          : "{0} cannot use any system reserved words, end with dots (.) or use any of the following characters: <code class='emphasized'>{1}</code>",
+    "CDO_INVALID_FILENAME_MESSAGE"      : "{0} cannot use any system reserved words, end with dots (.) or use any disallowed characters. Allowed characters: <code class='emphasized'>{1}</code>",
```

Comment:
nit: Since we're showing the allowed characters, not disallowed ones, I think it would be clearer to say "must use only allowed characters and cannot use any system reserved words or end with dots"

Reply:
yeah, good call -- it's a little roundabout right now